ibm/conns/config.go

```diff
@@ -3561,7 +3561,11 @@ func RefreshToken(sess *bxsession.Session) error {
 func EnvFallBack(envs []string, defaultValue string) string {
 	for _, k := range envs {
 		if v := os.Getenv(k); v != "" {
-			return v
+			if strings.Contains(v, "https://") {
+				return v
+			} else {
+				return fmt.Sprintf("https://%s/v1", v)
```

Comment:
why were these changes made?

Reply:
These changes were from the generated script. If not required, I can remove them

Reply:
yes remove it

Reply:
Done

Reply:
FYI: I opened up a PR in the generator that fixes this issue to avoid future confusion.